plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/data/MapboxViewportDataSourceOptions.kt

```diff
@@ -0,0 +1,307 @@
+package com.mapbox.maps.plugin.viewport.data
+
+import android.location.Location
+
+/**
+ * Options that impact generation of frames.
+ */
+class MapboxViewportDataSourceOptions internal constructor() {
```

Comment:
should be internal I guess

Reply:
applies to almost all the files in the PR, imho we should expose only user-facing interfaces, everything else should be hidden

Reply:
As mentioned in the PR's description. Currently the activity is accessing this class directly, we need to refactor and abstract the public interface from it in the follow-up PRs.